softwareComponents/configuration/include/configuration/rofibot.hpp

```diff
@@ -585,6 +589,24 @@ class Rofibot {
         return { true, "" };
     }
 
+    /**
+     * \brief Prepare configuration if needed and decide whether it is valid
+     *
+     * \return A pair - first item indicates the validity, the second one gives
+     * textual description of the reason for invalidity
+     */
+    template < typename Collision >
+    std::pair< bool, std::string > validate( Collision collisionModel = Collision() ) {
```

Comment:
Consider adding `[[nodiscard]]`. (So it's not used instead of `prepare`, because `prepare` can throw but this doesn't.

Reply:
Good point, added.